.github/workflows/add_opened_pr_to_project.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Create Project Card for a New PR'
```

Comment:
A new GitHub workflow that is run whenever a new PR is opened in this repo.  It simply executes a small script that adds this PR to the specified GitHub Project.